src/lib/CalibrationVerification.cpp

```diff
@@ -989,9 +998,17 @@ void CalibrationVerification::PrintErrorsSummary() {
   }
   norms_averaged = norms_summed / camera_errors_opt_.size();
 
+  float angular_summed, angular_averaged;
+  for (int i = 0; i < camera_angular_errors_.size(); i++) {
+    angular_summed += camera_angular_errors_[i];
+  }
+  angular_averaged = angular_summed / camera_angular_errors_.size();
```

Comment:
you should define variables the latest possible:
```suggestion
  float angular_averaged = angular_summed / camera_angular_errors_.size();
```